app/src/main/res/values/dimens.xml

```diff
@@ -3,4 +3,10 @@
     <dimen name="bottom_sheet_elevation">10dp</dimen>
     <dimen name="practice_icon_size">120dp</dimen>
     <dimen name="practice_test_elevation">4dp</dimen>
+
+    <dimen name="activity_horizontal_margin">16dp</dimen>
+    <dimen name="activity_vertical_margin">16dp</dimen>
+    <dimen name="fab_margin">16dp</dimen>
```

Comment:
Only fab_margin is being used, and not even for a fab, so these should be removed